.gitignore

```diff
@@ -4,3 +4,4 @@
 /composer.lock
 /docker-compose.yml
 /phpunit.xml
+.idea
```

Comment:
Please revert this too (see https://insight.symfony.com/projects/a7179df5-4ed7-468c-899c-891535dbe802/analyses/231?status=commit) and leave a blank line at the end of file.